zerver/lib/message.py

```diff
@@ -276,26 +272,39 @@ def sew_submessages_and_reactions_to_msgs(messages: List[Dict[str, Any]]) -> Lis
         return sew_messages_and_reactions(messages, reactions)
 
     @staticmethod
-    def to_dict_uncached_helper(message: Message) -> Dict[str, Any]:
-        return MessageDict.build_message_dict(
-            message = message,
-            message_id = message.id,
-            last_edit_time = message.last_edit_time,
-            edit_history = message.edit_history,
-            content = message.content,
-            topic_name = message.topic_name(),
-            date_sent = message.date_sent,
-            rendered_content = message.rendered_content,
-            rendered_content_version = message.rendered_content_version,
-            sender_id = message.sender.id,
-            sender_realm_id = message.sender.realm_id,
-            sending_client_name = message.sending_client.name,
-            recipient_id = message.recipient.id,
-            recipient_type = message.recipient.type,
-            recipient_type_id = message.recipient.type_id,
-            reactions = Reaction.get_raw_db_rows([message.id]),
-            submessages = SubMessage.get_raw_db_rows([message.id]),
-        )
+    def to_dict_uncached(messages: List[Message]) -> Dict[int, bytes]:
+        messages_dict = MessageDict.to_dict_uncached_helper(messages)
+        encoded_messages = {msg['id']: stringify_message_dict(msg) for msg in messages_dict}
+        return encoded_messages
+
+    @staticmethod
+    def to_dict_uncached_helper(messages: List[Message]) -> List[Dict[str, Any]]:
+        messages_dict = []
```

Comment:
I think you want to rename this `messages_list`.  Or maybe just change the helper to actually return a dict?

Reply:
As discussed on chat, `messages_dict` to `message_dicts` would probably suffice here.